src/terminal7.ts

```diff
@@ -916,8 +915,7 @@ export class Terminal7 {
         while (this.logBuffer.length > 0) {
             data += this.logBuffer.shift() + "\n"
         }
-        await Clipboard.write({string: data})
-        this.notify("Log copied to clipboard")
+        return data
         /* TODO: wwould be nice to store log to file, problme is 
```

Comment:
if you want to save to log to local pc or anywhere else
you need to do that on server side not client. thats why the module isn't working.

Reply:
@daonb

Reply:
We can send a API call to backend and it will do the rest for us

Reply:
Capacitor Filesystem API doesnt support PWA , so its should work only on IOS and Andriod